nest/src/auth/auth.controller.ts

```diff
@@ -1,15 +1,40 @@
-import { Controller, Get, Req, Res, UseGuards } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  HttpStatus,
+  Req,
+  Res,
+  UseGuards,
+} from '@nestjs/common';
 import { ApiTags } from '@nestjs/swagger';
 import { AuthService } from './auth.service';
 import { IntraGuard } from './guards/intra.guard';
 import * as process from 'process';
 import { Intra42User } from '../users/interface/intra42-user.interface';
+import { JwtRefreshGuard } from './guards/jwt-refresh.guard';
+import { JwtGuard } from './guards/jwt.guard';
 
 @Controller('auth')
 @ApiTags('auth')
 export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
+  private setCookieTokens(
+    tokens: { accessToken: string; refreshToken: string },
+    res: any,
+  ) {
+    res.cookie('access_token', tokens.accessToken, {
+      maxAge: 2592000000,
+      sameSite: true,
+      secure: false,
+    });
+    res.cookie('refresh_token', tokens.refreshToken, {
+      maxAge: 2592000000,
```

Comment:
Shouldn't the max age of the refresh token be greater than the one of the access token ?

Reply:
Actually the cookies should expire at the same time as the actual tokens do.